ui/tests/acceptance/secure-variables-test.js

```diff
@@ -119,6 +119,25 @@ module('Acceptance | secure variables', function (hooks) {
     assert.notOk(fooLink, 'foo0 file is no longer present');
   });
 
+  test('it does not allow you to save if you lack Items', async function (assert) {
+    defaultScenario(server);
+    window.localStorage.nomadTokenSecret = server.db.tokens[0].secretId;
```

Comment:
`question`: Should we clean this up at the end of the test?

Reply:
I think this probably isn't harmful / no need to clean it up yet. The precedent for this is found in another dozen+ test files throughout.